src/logger.js

```diff
@@ -2,15 +2,9 @@ import {
     transports,
     Logger as WinstonLogger,
 } from 'winston';
-
 import 'winston-loggly'; // adds winston.transports.Loggly
-import morgan from 'morgan'; // eslint-disable-line import/no-extraneous-dependencies
```

Comment:
Morgan is a express logging middleware we have a wrapper for. I moved all the logic related to the middleware into a separate middleware module.